examples/proxy/lib.rs

```diff
@@ -0,0 +1,71 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink_lang as ink;
+
+#[ink::contract]
+pub mod proxy {
+    /// A simple proxy contract, enabling upgradable smart contracts.
```

Comment:
We need to clarify that it is not a full upgradable contract because this contract only forwards calls to another contract. But the state is stored on a forwarded contract, not a Proxy. When delegated call will be implemented on `contract-pallet`, we should use it here, and then it will be a full upgradable contract.

Reply:
Good point, I clarified it. 

Btw.: The proxy contract is everything we've currently planned in the direction of upgradable contracts. I think the upgradable contracts milestone is out of scope for your current grant, but having proper delegate-call support in the `contracts` pallet as well as ink! would be a great follow-up to your current grant.